content/Store Support/Payments and Invoices/Find-your-receipt-or-invoice.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Find your receipt or invoice"
+id: 360016121859
+---
+
+When you make a purchase with us, your invoice or receipt is sent to the email address you provided at checkout.
+
+## Find your receipt or invoice
+
+Look for an email from us in your inbox. This email is sent after your purchase is processed. The sender details differ depending on whether you purchased from our EU or US store.
```

Comment:
```suggestion

Learn how to find a receipt or invoice for your purchase.

In this article:

* [Find the receipt or invoice for an Arduino Store purchase](#arduino-store)
* [Find the receipt or invoice for an Amazon purchase](#amazon)

## Find the receipt or invoice for an Arduino Store purchase {#arduino-store}

When you make a purchase in the [Arduino Store](https://store.arduino.cc/), your invoice or receipt is sent to the email address you provided at checkout.

The sender details differ depending on the region:
```
* The first sentence is good, but doesn't apply to Amazon. Moved it to the Arduino Store section.
* Also updated heading names to more clearly reflect the content of those sections.
* Added a TOC.
* Used the more customer-facing "region" choice instead of US/EU to distinguish between stores.